### PR TITLE
MDE: Feature multiselection cb

### PR DESCRIPTION
**Pull request by sukunis**
add multiselection combobox as available inputfield type (define type="CheckComboBox")

### CODE REVIEW

File: src/config/mdeConfiguration.xml [hunk 1]
```diff
@@ -30,6 +30,8 @@
                          Unit="mm" Value="millimeter value" Visible="true" />
                 <TagData DefaultValues="Value1,Value2,Value3" Name="Tag of Type ComboBox" Type="ComboBox"
                          Unit="" Value="Value1" Visible="true" />
+                <TagData DefaultValues="Value1,Value2,Value3" Name="Tag of Type CheckComboBox" Type="CheckComboBox"
```
Comment (dominikl) [inline src/config/mdeConfiguration.xml:33]: Just wondering if the Type should be changed to `CheckedComboBox` too?

File: src/config/mdeConfiguration.xml [hunk 2]
```diff
@@ -380,6 +382,8 @@
                          Unit="nm" Value="" Visible="true" />
                 <TagData DefaultValues="Value1,Value2,Value3" Name="Tag of Type ComboBox" Type="ComboBox"
                          Unit="" Value="Value1" Visible="true" />
+                <TagData DefaultValues="Value1,Value2,Value3" Name="Tag of Type CheckComboBox" Type="CheckComboBox"
```
Comment (dominikl) [inline src/config/mdeConfiguration.xml:385]: Same here.

File: src/config/mdeConfiguration.xml (hunk 1 above)
Comment (sukunis) [reply src/config/mdeConfiguration.xml:33]: To be honest: I don't quite understand what you mean.

Comment (dominikl) [reply src/config/mdeConfiguration.xml:33]: I was wondering if this should match the class name, which is `CheckedComboBox.java`.

**Comment by dominikl** [comment]
👍 Looks good to me.

**Pull request merged**